sentry_sdk/integrations/opentelemetry/integration.py

```diff
@@ -46,9 +51,14 @@ def setup_once():
 
 def _setup_sentry_tracing():
     # type: () -> None
+    import opentelemetry.context
+
+    opentelemetry.context._RUNTIME_CONTEXT = SentryContextVarsRuntimeContext()
+
     provider = TracerProvider()
-    provider.add_span_processor(SentrySpanProcessor())
+    provider.add_span_processor(PotelSentrySpanProcessor())
```

Comment:
Do we plan to include `Potel` in the final name of this class before it goes out to users? I am guessing our users will not know what "Potel" is

Reply:
Since we're dropping the old span processor we can eventually just go with `SentrySpanProcessor`.